### PR TITLE
Fix two typos and ease curl copy pasta

### PR DESCRIPTION
**Pull request by michaeladda**

### CODE REVIEW

File: five-minute-serverless-backend/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Five Minute Serverless Backend
-*How to build a **serverless backend** for your web applciation and have it up in running on the cloud in minutes.*
+# Five Minutes Serverless Backend
```
Comment (rylandg) [inline five-minute-serverless-backend/README.md]: It's actually correct as is. Using the singular is valid when the thing being described is singular "Serverless Backend". Here is a coincidentally perfect reference https://ell.stackexchange.com/questions/14290/shouldnt-five-minute-walk-be-five-minutes-walk-in-this-sentence

File: five-minute-serverless-backend/README.md [hunk 2]
```diff
@@ -58,11 +58,11 @@ Invoke with one of:
 Our function is up and running and will autoscale with invocations. We can use `curl` to invoke it:
 
 ```
-$ curl https://run.binaris.com/v2/run/<account-id>/public_backend -d '"hello"'
+$ curl https://run.binaris.com/v2/run/$(bn show accountId)/public_backend -d '"hello"'
 "HELLO"
 ```
 
-Make sure to replace `<account-id>` with your own account ID. If you don't have it handy, you can find that by typing
+In the actual deploy printout `<account-id>` will be replaced by your own account ID. If you don't have it handy, you can find that by typing
```
Comment (rylandg) [inline five-minute-serverless-backend/README.md]: Why do you keep this if the actual curl line no longer has it?

**Comment by michaeladda** [comment]
how is there a conflict? @avnerbraverman did you just push some of the changes discussed here directly to master without a PR?!?
I see https://github.com/binaris/functions-examples/commit/56ded0ee4917d36a6171a7497d864a12ba27e670
yet https://github.com/binaris/functions-examples/pulls?utf8=%E2%9C%93&q=is%3Apr+ has no record of it. This is not how we handle code. It must be reviewed first.

**Comment by michaeladda** [comment]
@rylandg please take another look

**Pull request merged**